### ISSUE TITLE
Classification loss calculated incorrectly

### ISSUE THREAD

**Issue by luisbro** [atomgptlab/alignn#1]
Hello, I am trying to fine-tune alignn for predicting a custom property and used the classification example for that. However, even in the example notebook, it doesn't seem to learn anything and predicts only label zero with a loss that is negative. After looking into it, I think the issue stems from the output of the alignn atomwise model, which outputs a single sigmoid value. The sigmoid value is then used to calculate a `NLLLoss` (expecting a list of log softmax for each target), instead of something like a `BCELoss`.

It seems like changing this
https://github.com/atomgptlab/alignn/blob/408bb6e996006dc4441ea48d4189047742dad637/alignn/models/alignn_atomwise.py#L345-L348

to something like
```
        if self.classification:
            self.fc = nn.Linear(config.hidden_features, 2)
            # self.softmax = nn.Sigmoid()
            self.softmax = nn.LogSoftmax(dim=-1)
```
should work and is essentially what was changed in [this commit](https://github.com/atomgptlab/alignn/commit/809dedf5e01eafb34b428bc006c571eef450a8c3#diff-0b6f86bc2013f05255408c1955d8a79bd8fdfa827d9112c397f6d9c03e89455d). A small difference is the dim=-1 which avoids another error.

I'm not sure where else this change could affect something, so I'd leave it here as a suggestion and help for anyone else running into this for now. Or is there even another way to fine-tune alignn for property predictions, that is more recent, explaining why this bug hasn't been a problem until now?

**Comment by luisbro** [comment]
This seems to have been addressed in 76c56ebe8f3e83a9144bb08250062a77a2f1eac6 and 7f99fa1973231f6749ba2c9cb5cd8c1ebebd299a, thanks (:
